src/app/ApplicationTemplate.Shared/Presentation/Posts/EditPostPageViewModel.cs

```diff
@@ -8,16 +8,20 @@
 using Chinook.StackNavigation;
 using MessageDialogService;
 using Microsoft.Extensions.Localization;
+using Uno;
 
 namespace ApplicationTemplate.Presentation
 {
-	public class EditPostPageViewModel : ViewModel
+	public partial class EditPostPageViewModel : ViewModel
```

Comment:
Do other viewmodels use the inject generation? If not, let's stay consistent for now.